### ISSUE TITLE
Toolbar View Selectors display as if active always

### ISSUE THREAD

**Issue by jeff-phillips-18** [patternfly/angular-patternfly#348]
Changes to patternfly styling for https://github.com/patternfly/patternfly/pull/440 broke the styling for the view selector buttons.

**Comment by bleathem** [comment]
This visible here:
http://www.patternfly.org/angular-patternfly/#/api/patternfly.toolbars.directive:pfToolbar#example_demo

The view selectors are always highlighted.

**Comment by bleathem** [comment]
The _active_ class is set via the following expression:
```
ng-class="{'active': isViewSelected(view.id), 'disabled': checkViewDisabled(view)}"
```

**Comment by bleathem** [comment]
The _active_ class toggling is working well.  The problem is caused by the upstream patternfly markup change.  Angular-patternfly will have to be update to use `<button>` elements rather than `<li>` elements.

**Comment by jeff-phillips-18** [comment]
Fixed by #360
